src/fs/rock/RockRebuild.cc

```diff
@@ -146,7 +193,7 @@ class LoadingSlot
 class LoadingParts
 {
 public:
-    LoadingParts(int dbSlotLimit, int dbEntryLimit);
+    LoadingParts(const char *dirPath);
```

Comment:
```suggestion
    explicit LoadingParts(const char *dirPath);
```
Or we would have to document why it is not.